src/cljs/harja/views/urakka/kulut/mhu_kustannusten_seuranta.cljs

```diff
@@ -40,11 +40,11 @@
 ; spekseistä laskettu
 (def leveydet {:caret-paaryhma "2%"
                :paaryhma-vari "2%"
-               :tehtava "41%"
```

Comment:
Nää prosenstit sinänsä ihan OK, mutta jos jättää % merkin pois, taulukko osaa muodostaa itse oikeat prosentit ihan suhteellisten leveyksien pohjalta. Tällöin siis sarakkeiden leveyksien summan ei tarvitsisi olla 100%.